R/migrations.R

```diff
@@ -23,7 +23,7 @@ run_migration <- function(queue, output_dir, dry_run = FALSE) {
 migrate_task <- function(task_id, queue, dry_run = FALSE) {
   message(sprintf("Migrating %s", task_id))
   res <- queue$queue$task_result(task_id)
-  if (!is.null(res$version) && identical(as.character(res$version), "2.5.0")) {
+  if (!is.null(res$version) && as.character(res$version) >= "2.5.0") {
```

Comment:
```suggestion
  if (!is.null(res$version) && numeric_version(res$version) >= "2.5.0") {
```

though curious what format the input has here?

Reply:
It should just be a character or NULL, so `as.character` cast is unnecessary